README.md

```diff
@@ -4,6 +4,7 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/eyalroz/printf/master/LICENSE)
 [![Github Bug-type issues](https://shields.io/github/issues-search/eyalroz/printf?query=is:open%20is:issue%20label:bug&label=open%20bugs)](http://github.com/eyalroz/printf/issues)
 <sup>Parent repository: </sup>[![Github issues (original repo)](https://img.shields.io/github/issues/mpaland/printf.svg)](http://github.com/mpaland/printf/issues)
+<sup>Package managers: </sup>[![hunter](https://img.shields.io/badge/hunter-eyalroz_printf-blue.svg)](https://github.com/cpp-pm/hunter/blob/master/examples/eyalroz_printf/CMakeLists.txt)
```

Comment:
If we are to add shields for package managers, it has to be the other ones as well, i.e. conan and vcpkg. I don't think it's reasonable to promote just one over the others (although I'm sure you guys like yours best...)

Reply:
I didn't find `printf` support in these PMs:
https://conan.io/center/recipes?value=printf
https://vcpkg.io/en/packages
I think that when someone adds package in PM they will be able to add the new badge. The main thing is to show with an example where to add it :smiley:

Reply:
Oh, sorry, I'm mixing up my own libraries. I have another package there, cuda-api-wrappers.